clients/src/main/resources/common/message/BeginQuorumEpochResponse.json

```diff
@@ -17,25 +17,35 @@
   "apiKey": 53,
   "type": "response",
   "name": "BeginQuorumEpochResponse",
-  "validVersions": "0",
-  "flexibleVersions": "none",
+  "validVersions": "0-1",
```

Comment:
Please add a comment to mention what changed in v.1.

Reply:
Fixed.